src/status_im/setup/build_hooks/worklets.clj

```diff
@@ -0,0 +1,65 @@
+(ns status-im.setup.build-hooks.worklets
+  (:require [clojure.set])
+  (:import (java.io BufferedReader InputStreamReader PrintWriter)))
+
+(defn- update-js-output
+  [build-state code-processed]
+  (reduce-kv (fn [prev-build-state ns-key new-code]
+               (assoc-in prev-build-state [:output ns-key :js] new-code))
+             build-state
+             code-processed))
+
+(defn- get-workletized-code!
+  [code-seq store-atom]
+  (doall
+   (pmap (fn [[[_ filepath :as ns-key] js-code]]
+           (println "Workletizing:" filepath) ;; TODO: debug remove
+           (try
+             (let [command       "node workletize-code.js"
+                   process       (.exec (Runtime/getRuntime) command)
+                   output-stream (.getOutputStream process)
+                   writer        (PrintWriter. output-stream)
+                   reader        (BufferedReader. (InputStreamReader. (.getInputStream process)))]
+               (.println writer js-code)
+               (.flush writer)
+               (.close writer)
+
+               (let [output     (apply str (interleave (line-seq reader) (repeat "\n")))
+                     _exit-code (.waitFor process)]
```

Comment:
I identified the problem, while running `make release-android` this process returns a `exit-code` of `1`, so the code transformed is just `""`.

While executing `make run-clojure` its exit code is `0` and we actually have the result.

What might be the reason?
I need help of someone familiar with our build process.

Reply:
But the command I'm executing is to create a node subprocess, that's the one with 0 and 1 exit codes